core/domain/collection_validators.py

```diff
@@ -0,0 +1,485 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validators for collection models."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+import itertools
+import re
+
+from constants import constants
+from core.domain import base_model_validators
+from core.domain import classifier_domain
+from core.domain import classifier_services
+from core.domain import collection_domain
+from core.domain import collection_services
+from core.domain import config_domain
+from core.domain import exp_domain
+from core.domain import exp_fetchers
+from core.domain import exp_services
+from core.domain import learner_progress_services
+from core.domain import platform_parameter_domain
+from core.domain import question_domain
+from core.domain import question_fetchers
+from core.domain import question_services
+from core.domain import recommendations_services
+from core.domain import rights_domain
+from core.domain import rights_manager
+from core.domain import skill_domain
+from core.domain import skill_fetchers
+from core.domain import skill_services
+from core.domain import stats_services
+from core.domain import story_domain
+from core.domain import story_fetchers
+from core.domain import subtopic_page_domain
+from core.domain import subtopic_page_services
+from core.domain import suggestion_services
+from core.domain import topic_domain
+from core.domain import topic_fetchers
+from core.domain import topic_services
+from core.domain import user_domain
+from core.domain import user_services
+from core.domain import voiceover_services
+from core.domain import wipeout_service
+from core.platform import models
+import feconf
+import python_utils
+import utils
+
+(
+    base_models, collection_models, config_models,
+    email_models, exp_models, feedback_models,
```

Comment:
Is `email_models ` used  any this collection model?

Reply:
I don't think so, can you tell how can I edit it in this pull request? or will it be raised as a new issue?

Reply:
You can make changes in the same branch(`EishaMazhar:eisha-develop`) and commit and push the changes. All the changes made from the same branch would get updated to the same PR.